framework/src/modules/chain/rounds/rounds.js

```diff
@@ -48,11 +46,14 @@ class Rounds {
 		library = {
```

Comment:
why do we keep `library` on this file?

Reply:
As commented above

Reply:
Can we create an issue to make the refactor in separate PR?

Reply:
This is being rewritten in the BFT branch